README.md

```diff
@@ -35,34 +43,55 @@ To make your kernel compatible with `bootloader`:
     };
     bootloader_api::entry_point!(kernel_main, config = &CONFIG);
     ```
-- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` before to download precompiled versions of the `core` and `alloc` crates.
+- Compile your kernel to an ELF executable by running **`cargo build --target x86_64-unknown-none`**. You might need to run `rustup target add x86_64-unknown-none` before to download precompiled versions of the `core` and `alloc` crates. You can add `x86_64-unknown-none` as default target and add it to your toolchain so that `cargo build` takes care of this.
+    ```toml
+    # .cargo/config.toml
+    [build]
+    target = "x86_64-unknown-none"
+    ```
+    ```sh
+    $ cargo build
+    ```
 - Thanks to the `entry_point` macro, the compiled executable contains a special section with metadata and the serialized config, which will enable the `bootloader` crate to load it.
 
-### Booting
-
-To combine your kernel with a bootloader and create a bootable disk image, follow these steps:
+#### Creating a bootable image
 
 - Move your full kernel code into a `kernel` subdirectory.
-- Create a new `os` crate at the top level that defines a [workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html).
-- Add a `build-dependencies` on the `bootloader` crate.
-- Create a [`build.rs`](https://doc.rust-lang.org/cargo/reference/build-scripts.html) build script.
-- Set up an [artifact dependency](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#artifact-dependencies) to add your `kernel` crate as a `build-dependency`:
-  ```toml
-  # in Cargo.toml
-  [build-dependencies]
-  kernel = { path = "kernel", artifact = "bin", target = "x86_64-unknown-none" }
-  ```
-  ```toml
-  # .cargo/config.toml
-
-  [unstable]
-  # enable the unstable artifact-dependencies feature, see
-  # https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#artifact-dependencies
-  bindeps = true
-  ```
-  Alternatively, you can use [`std::process::Command`](https://doc.rust-lang.org/stable/std/process/struct.Command.html) to invoke the build command of your kernel in the `build.rs` script. 
-- Obtain the path to the kernel executable. When using an artifact dependency, you can retrieve this path using `std::env::var_os("CARGO_BIN_FILE_MY_KERNEL_my-kernel")`
-- Use `bootloader::UefiBoot` and/or `bootloader::BiosBoot` to create a bootable disk image with your kernel.
+- Create a new `os` crate at the top level
+    ```sh
+    $ cargo init --bin
+    ```
+- Define a [workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html) and add your kernel as a workspace member.
```

Comment:
If we're specifying the command used to create the top-level crate (`cargo init --bin` we should do the same for the kernel). We could remove the kernel from the members (and just use `members = []`) and then use `cargo new kernel --bin` in the workspace root.